clients/fossid-webapp/src/main/kotlin/model/status/ScanDescription2021dot2.kt

```diff
@@ -19,11 +19,13 @@
 
 package org.ossreviewtoolkit.clients.fossid.model.status
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
 
 /**
  * A description of scan status. This class is for FossID version 2021.2.
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Why do properties also need to be ignored for old version 2021.2, when the commit message says the new property was introduced in version (20)23.2? Or is this class being used *starting* with version 2021.2 and onward?

Maybe this is also something that could be clarified in the class description while at it: Add a commit that documents `ScanDescription` and `ScanDescription2021dot2` more clearly with regard to which versions they're used with.

Reply:
Ok, found it, `ScanDescription2021dot2` is really being used with version 2021.2 *and onward*:

https://github.com/oss-review-toolkit/ort/blob/3b318d687c0c1aeb7fd790a961f56aeb829ad1f5/clients/fossid-webapp/src/main/kotlin/FossIdServiceWithVersion.kt#L36

Then I'd have the opposite question: Why ignore properties also in the (legacy) `ScanDescription`?

Reply:
You're right, I removed it from `ScanDescription`.